src/my_crypto.erl

```diff
@@ -19,17 +19,38 @@ is_public_key(Key) ->
 
 %% Actual function implementations
 
-hash_full(Bin) -> crypto:hash(sha256, Bin).
+hash_full(Bin) -> 
+    Full = crypto:hash(sha256, Bin),
 
-sign_full(Hash, PrivateKey) -> public_key:sign(Hash, none, PrivateKey).
+    %% Copied this online :cat_scream:
+    HexString = lists:flatten([io_lib:format("~2.16.0B",[X]) || <<X:8>> <= Full ]),
 
-verify_full(Hash, Signature, PubKey) -> public_key:verify(Hash, none, Signature, PubKey).
+    list_to_binary(HexString).
```

Comment:
so what type is Full 😱?

Reply:
I guess I could look it up....

Reply:
this returns a binary string. needed it this way instead of regular string for json compatibility cause erlang is stupid about strings

Reply:
oh, and to your question, Full is binary. I just translate bytes to hex notation like 3F, A1 etc